### PR TITLE
added history page

### PR DESCRIPTION
**Pull request by ashishnnnnn**
-> User can see their updated history in history page.
-> User can delete all the data from history.
-> User can delete single history.
-> If user visit same video which is in history then the video will be added as latest history and past one deleted.
-> On History page , the last watched video will appear first and second last is on second and so on...

### CODE REVIEW

File: src/ApiCalls/removeAllHistory/removeAllHistory.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        type: "REMOVE_ALL_HISTORY",
+        payload: response.data.history,
+      });
+      handleaddtoast({
+        message: `Deleted the History`,
+        type: "alert-success",
+      });
+    }
+  } catch (e) {
+    console.log("error occured: ", e);
```
Comment (SukanyaSen530) [inline src/ApiCalls/removeAllHistory/removeAllHistory.js]: You can remove the console.log if you are using toast! :)

**Review by SukanyaSen530 — APPROVED** [review]
I like that you are using the same video card for showing history as well!

**Review by tushartiwari7 — APPROVED** [review]
Nice, Liked that you have implemented auth in video library app too.
Overall Code Quality is also maintained.
Good to merge.

**Review by shubhamkumar648 — APPROVED** [review]
I like the  way you use the variable name ,code looks neat and clean ,good to go

**Pull request merged**